Mana-Cores/traits.json

```diff
@@ -201,8 +194,6 @@
 		"valid": false,
 		"purifiable": false,
 		"player_display": true,
-		"mana_multiplier": 8,
-		"flags": [ "SILENT_SPELL", "SUBTLE_SPELL" ],

```

Comment:
I think the silent and subtle spell was accidently removed.

Reply:
So it was, fixed.